R/load_files.R

```diff
@@ -904,18 +910,19 @@ get_df_from_raw_edited_EnVision_df <-
     do.call(rbind, res_l)
```

Comment:
I suggest switching to data.table::rbindlist since res_1 is a list of data.tables